spec/poros/food_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Food do
+  it 'can populate food fields from JSON response' do
+    data = {
+      'description': 'Cheddar Cheese',
+      'gtinUPC': "8901020020844",
+      'brandOwner': 'Kellog',
+      'ingredients': 'cheese'
+    }
+
+    food = Food.new(data)
```

Comment:
This looks good. Since this is the PORO spec, I think you could also have your expects be specific like:
```
expect(food.description).to eq('Cheddar Cheese')
```